CollAction/Controllers/AccountController.cs

```diff
@@ -328,6 +328,54 @@ public IActionResult ResetPasswordConfirmation()
             return View();
         }
 
+        [HttpGet]
+        [AllowAnonymous]
+        public IActionResult FinishRegistration(string email, string code)
+        {
+            var model = new FinishRegistrationViewModel { Email = email, Code = code };
```

Comment:
If you put the model directly in the parameter list (instead of string email, string code), you can skip this line

Reply:
Unfortunately, it will conflict with the POST method that has the same name.